osu.Game/Rulesets/Scoring/DrainingHealthProcessor.cs

```diff
@@ -47,6 +47,8 @@ public class DrainingHealthProcessor : HealthProcessor
         private double targetMinimumHealth;
         private double drainRate = 1;
 
+        private int breakendcounter;
```

Comment:
Why is this an integer? It's only ever incremented and checked if above zero. Could be bool instead.

Reply:
It is so that I can keep track of Hits using `breakendcounter++;`

Reply:
That's what I said tho. You only write to this value by resetting it or incrementing it. And you only read from this value by seeing if it has been incremented. idk how is it not clear that you can make this a boolean instead.

Reply:
Fixed in 61be2f5.